scripts/generator.py

```diff
@@ -118,6 +123,10 @@ def argument_parser() -> argparse.Namespace:
                         help='enforce strict checking at schema cleanup')
     parser.add_argument('--intermediate-only', action='store_true',
                         help='generate intermediary files only')
+    parser.add_argument('--no-ecs', action='store_true',
```

Comment:
Could add more documentation to `help` for the new arguments, to explain more about how these are intended to be used, and to mention they are for custom use cases. It's fine to use multiline strings here, so the help message can be longer.